contrib/build-linux/appimage/build.sh

```diff
@@ -55,44 +49,31 @@ $SUDO docker build -t $IMGNAME \
     contrib/build-linux/appimage \
     || fail "Failed to create docker image"
 
-# This is the place where we checkout and put the exact revision we want to work
-# on. Docker will run mapping this directory to /opt/electrumabc
-FRESH_CLONE=`pwd`/contrib/build-linux/fresh_clone
-FRESH_CLONE_DIR=$FRESH_CLONE/$GIT_DIR_NAME
+PROJECT_ROOT=`pwd`
 MAPPED_DIR=/opt/electrumabc
 
-(
-    $SUDO rm -fr $FRESH_CLONE && \
-        mkdir -p $FRESH_CLONE && \
-        cd $FRESH_CLONE  && \
-        git clone $GIT_REPO && \
-        cd $GIT_DIR_NAME && \
-        git checkout $REV
-) || fail "Could not create a fresh clone from git"
-
-mkdir "$FRESH_CLONE_DIR/contrib/build-linux/home" || fail "Failed to create home directory"
+mkdir "${PROJECT_ROOT}/contrib/build-linux/home" || fail "Failed to create home directory"
 
 (
     $SUDO docker run $DOCKER_RUN_TTY \
     -e HOME="$MAPPED_DIR/contrib/build-linux/home" \
     -e BUILD_DEBUG="$BUILD_DEBUG" \
     --name $CONTAINERNAME \
-    -v $FRESH_CLONE_DIR:$MAPPED_DIR:delegated \
+    -v ${PROJECT_ROOT}:$MAPPED_DIR:delegated \
     --rm \
     --workdir $MAPPED_DIR/contrib/build-linux/appimage \
     -u $(id -u $USER):$(id -g $USER) \
     $IMGNAME \
-    ./_build.sh $REV
+    ./_build.sh
 ) || fail "Build inside docker container failed"
 
 popd
 
 info "Copying built files out of working clone..."
 mkdir -p dist/
-cp -fpvR $FRESH_CLONE_DIR/dist/* dist/ || fail "Could not copy files"
 
-info "Removing $FRESH_CLONE ..."
-$SUDO rm -fr $FRESH_CLONE
+info "Removing HOME ..."
```

Comment:
oO
If I see that log on my script you can be sure I run CTRL+C compulsively the nanosecond after I saw it